Makefile.toml

```diff
@@ -1,7 +1,22 @@
+extend = [
+    { path = "./make/ci.toml"}
+]
+
+[config]
+default_to_workspace = false
```

Comment:
Does this mean we can remove all the mentions of workspace in all the rules? Neat.